desktop-src/dlgbox/dlgbox-programming-considerations.md

```diff
@@ -27,33 +27,13 @@ ms.date: 05/31/2018
 
 This overview discusses some programming considerations concerning dialog boxes.
 
-The overview includes the following topics.
-
--   [Dialog Box Procedures](#dialog-box-procedures)
-    -   [The WM\_INITDIALOG Message](https://docs.microsoft.com/windows)
-    -   [The WM\_COMMAND Message](https://docs.microsoft.com/windows)
-    -   [The WM\_PARENTNOTIFY Message](https://docs.microsoft.com/windows)
-    -   [Control-Color Messages](#control-color-messages)
-    -   [Dialog Box Default Message Processing](#dialog-box-default-message-processing)
--   [Dialog Box Keyboard Interface](#dialog-box-keyboard-interface)
-    -   [The WS\_TABSTOP Style](https://docs.microsoft.com/windows)
-    -   [The WS\_GROUP Style](https://docs.microsoft.com/windows)
-    -   [Mnemonics](#mnemonics)
--   [Dialog Box Settings](#dialog-box-settings)
-    -   [Radio Buttons and Check Boxes](#radio-buttons-and-check-boxes)
-    -   [Dialog Box Edit Controls](#dialog-box-edit-controls)
-    -   [List Boxes, Combo Boxes, and Directory Listings](#list-boxes-combo-boxes-and-directory-listings)
-    -   [Dialog Box Control Messages](#dialog-box-control-messages)
--   [Custom Dialog Boxes](#custom-dialog-boxes)
-
```

Comment:
I've removed this part because there is already a sidebar TOC that shows all the `H2`s

Reply:
You are correct, but this section facilitates users clicking directly to what they need instead of skimming a TOC and guessing which section might contain the content they're looking for. Please replace this and I will approve the PR.